reference/supervisor.md

```diff
@@ -383,18 +384,28 @@ const char *wb_supervisor_node_get_base_type_name(WbNodeRef node);
 namespace webots {
   class Node {
     enum {
-      NO_NODE, ACCELEROMETER, APPEARANCE, BACKGROUND, BALL_JOINT, BALL_JOINT_PARAMETERS,
-      BOX, BRAKE, CAMERA, CAPSULE, CHARGER, COLOR, COMPASS, CONE, CONNECTOR, CONTACT_PROPERTIES,
-      COORDINATE, CYLINDER, DAMPING, DIFFERENTIAL_WHEELS, DIRECTIONAL_LIGHT, DISPLAY,
-      DISTANCE_SENSOR, ELEVATION_GRID, EMITTER, EXTRUSION, FOCUS, FLUID, FOG, GPS,
-      GROUP, GYRO, HINGE_2_JOINT, HINGE_2_JOINT_PARAMETERS, HINGE_JOINT, HINGE_JOINT_PARAMETERS,
-      IMAGE_TEXTURE, IMMERSION_PROPERTIES, INDEXED_FACE_SET, INDEXED_LINE_SET, INERTIAL_UNIT,
-      JOINT_PARAMETERS, LED, LENS_DISTORTION, LIDAR, LIGHT_SENSOR, LINEAR_MOTOR,
-      MATERIAL, MICROPHONE, PEN, PHYSICS, PLANE, POINT_LIGHT, POSITION_SENSOR, PROPELLER,
-      RADAR, RADIO, RANGE_FINDER, RECEIVER, RECOGNITION, ROBOT, ROTATIONAL_MOTOR,
-      SHAPE, SLIDER_JOINT, SLOT, SOLID, SOLID_REFERENCE, SPEAKER, SPHERE, SPOT_LIGHT,
-      SUPERVISOR, SWITCH, TEXTURE_COORDINATE, TEXTURE_TRANSFORM, TOUCH_SENSOR, TRACK,
-      TRACK_WHEEL, TRANSFORM, VIEWPOINT, WORLD_INFO, ZOOM
+      NO_NODE,
+      // VRML97 nodes
```

Comment:
Does the user really needs to know which nodes are VRML nodes, etc?
Personally I think that we should list all of them in alphabetical order as before and that this comments are not useful for the users.

If we really want to split them in groups then we should better choose them.

Reply:
My concern is more about to match what we write in nodes.h matches directly with the doc to facilitate maintenance. A big list in alphabetical order would be indeed simpler to understand and to maintain, but we should apply the same in nodes.h.

I propose to accept this as-is (it has sense in term of consistency with the header), and to switch to a big list in alphabetical order in webots-doc@webots-r2018a-rev3 + webots@master in later PRs.

Reply:
If we plan to change the list, then the maintenance doesn't seem a problem because we won't need to maintain it .
Note that nowhere in the documentation other than here we describe which nodes are VRML97 nodes. So it doesn't seem good to introduce concepts in the documentation that are not explained nor needed and it doesn't seem very consistent within the documentation itself.

Reply:
VRML nodes: https://www.cyberbotics.com/doc/reference/node-chart

Reply:
Yes, but in the description for example of the Appearance node this is not mentioned.
So it doesn't seem that in our documentation the fact that Appearance is a VRML node is an important information. So why should be list the nodes here based on the fact that they are or not VRML nodes?
For example Box is also a geometry, and it would much more useful for a user to group all the geometries together rather than listing them in this way.
Code and documentation are different. Here we want to have a meaningful and clear list of nodes.  But at the moment it is confusing.

Reply:
> VRML nodes: https://www.cyberbotics.com/doc/reference/node-chart

It's a pity we wrote VRML instead of VRML97 on this chart...

Reply:
Note: I am not sure what is the best to present this list... It seems good to me to separate robots, devices and 3D geometry as it is the case in this PR (we could even add a "physics" category that would make the "misc" category smaller).

Reply:
> It's a pity we wrote VRML instead of VRML97 on this chart

It's not so a pity, now that we have mermaid it is a 10sec fix, i will do this right now.